src/app/authentication/layouts/default/default.component.scss

```diff
@@ -0,0 +1,49 @@
+@import '~bootstrap/scss/bootstrap';
+@import 'functions/sizing';
+@import 'variables';
+
+.content-wrapper {
+  display: flex;
+  height: 100vh;
+  flex-direction: column;
+  justify-content: center;
+
+  background-image: url('/assets/images/background.png');
+  background-position: center;
+  background-size: cover;
+
+  @include media-breakpoint-up(md) {
+    padding: 0;
+  }
+}
+
+.app-header {
+  &__logo {
+    height: rem(40px);
+  }
+
+  &__page-desc {
+    margin-top: rem(24px);
+    margin-bottom: 0;
+
+    color: map-get($grays, 600);
+  }
+}
+
+.app-navigation {
+  position: absolute;
+  top: 0;
+  margin-top: rem(15px);
+
+  @include media-breakpoint-up(md) {
+    margin: rem(32px 0 0 32px);
```

Comment:
I think it should be `margin: rem(37px 0 0 39px);` based on Based on design.

Reply:
@junan I think it's `32px` 🤔 
![Screen Shot 2564-01-14 at 11 20 25](https://user-images.githubusercontent.com/1772999/104544703-fe64db00-565a-11eb-9f59-dcf41eee597e.png)